yellowbrick/model_selection/CVScores.py

```diff
@@ -0,0 +1,174 @@
+
+# coding: utf-8
+
+# In[ ]:
+
+# yellowbrick.model_selection.CVScores
+#
+#
+# Author:   Prema Damodaran Roman
+
+#
+# Copyright (C) 2018 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: CVScores.py [7f47800] pdamo24@gmail.com $
```

Comment:
I've been thinking about the name of this file; in the scikit-learn repo, this module is simply called `sklearn.model_selection._validation`; what would you think about the more pythonic `cv.py` for this file? The `CVScores` visualizer would be imported in `__init__.py` -- another thing I'd recommend adding ;-)

Reply:
@bbengfort cv.py sounds good to me.  Could you explain what I need to do in the __init__.py code?

Reply:
this is what I added to __init__.py - is this correct (after making the change to call the file cv.py)?

from .cv import CVScores, cv

Reply:
that looks right to me!  @bbengfort

Reply:
Agreed, that's correct - but I don't see any changes to `__init__.py` -- did you push it?

Reply:
not yet - i wanted to confirm before i push